core/src/main/java/org/jdbi/v3/core/statement/SqlStatements.java

```diff
@@ -58,7 +58,9 @@ private SqlStatements(SqlStatements that) {
      * @return this
      */
     public SqlStatements define(String key, Object value) {
-        attributes.put(key, value);
+        if (value != null) {
+            attributes.put(key, value);
+        }
```

Comment:
Should probably `remove(key)` if `value == null`.

Reply:
good idea, thank you.